examples/create_stem_mpi.py

```diff
@@ -0,0 +1,81 @@
+import glob
+from stempy import io, image
+import numpy as np
+from mpi4py import MPI
+import sys
+from pathlib import Path
+import matplotlib.pyplot as plt
+import click
+
+def get_files(files):
+    comm = MPI.COMM_WORLD
+    rank = comm.Get_rank()
+    world_size = comm.Get_size()
+
+    return files[rank::world_size]
+
+@click.command()
+@click.argument('files', nargs=-1, type=click.Path(exists=True, dir_okay=False))
+@click.option('-c', '--center', help='The center (comma separated)', required=True)
+@click.option('-i', '--inner-radii', help='The inner radii (comma separated)', required=True)
+@click.option('-o', '--outer-radii', help='The outer radii (comma separated)', required=True)
+def main(files, center, inner_radii, outer_radii):
+    center = center.split(',')
+    if len(center) != 2:
+        raise click.ClickException('Center must be of the form: center_x,center_y.')
+
+    center_x, center_y = [int(x) for x in center]
+
+    inner_radii = inner_radii.split(',')
+    outer_radii = outer_radii.split(',')
+
+    if len(inner_radii) != len(outer_radii):
+        raise click.ClickException('Number of inner and outer radii must match')
+
+    inner_radii = [int(x) for x in inner_radii]
+    outer_radii = [int(x) for x in outer_radii]
+
+    comm = MPI.COMM_WORLD
+    rank = comm.Get_rank()
+    world_size = comm.Get_size()
+    comm.Barrier()
+    start = MPI.Wtime()
+
+    files = get_files(files)
+
+    # Create local sum
+    reader = io.reader(files, version=io.FileVersion.VERSION3)
+
+    # Get the scan image size
+    block = reader.read()
```

Comment:
By the way, if this ends up taking a lot of time, maybe in the future we can add a function to the `StreamReader` to just read a header and return it, rather than reading the whole block.

Reply:
Not a bad idea